force-app/main/default/lwc/propertyLocation/propertyLocation.js

```diff
@@ -0,0 +1,93 @@
+import { LightningElement, wire, api } from 'lwc';
+import { getLocationService } from 'lightning/mobileCapabilities';
+import { getRecord, getFieldValue } from 'lightning/uiRecordApi';
+
+// Using hardcoded fields due to LWC bug
+const LATITUDE_FIELD = 'Property__c.Location__Latitude__s';
+const LONGITUDE_FIELD = 'Property__c.Location__Longitude__s';
+
+const fields = [LATITUDE_FIELD, LONGITUDE_FIELD];
+
+export default class PropertyLocation extends LightningElement {
+    error;
+    myDeviceLocationService;
```

Comment:
Same remark as other recipe can you rename `myDeviceLocationService` to `deviceLocationService` for naming consistency?